post/release-0-2.qmd

```diff
@@ -14,39 +21,123 @@ since the last specification release.
 
 The GeoArrow specification establishes conventions for representing spatial data
 in [Apache Arrow](https://arrow.apache.org) formats (e.g., C Data Interface,
-Serialized IPC) and implementations (e.g., PyArrow, Arrow C++, arrow-rs).
+Serialized IPC) and implementations (e.g., [PyArrow][pyarrow], [Arrow C++][arrow-cpp], [arrow-rs]).
 It is separated into two parts:
 
-- A set of [extension type](https://geoarrow.org/extension-types.html) definitions
-  to propagate type-level spatial metadata (e.g., coordinate reference system, edge interpolation)
-- A set of [memory layouts](https://geoarrow.org/format.html) to represent geometry
+[pyarrow]: https://arrow.apache.org/docs/python/index.html
+[arrow-cpp]: https://arrow.apache.org/docs/cpp/index.html
+[arrow-rs]: https://docs.rs/arrow/latest/arrow/
+
+- A set of [memory layouts](../format.html) to represent geometry
```

Comment:
I think the memory layouts are the first thing someone thinks about when someone thinks about GeoArrow and so it makes sense to order this bullet point first